webapp/app.py

```diff
@@ -63,9 +61,14 @@ def login_required(func):
 
     def is_user_logged_in(*args, **kwargs):
```

Comment:
I'm confused. I thought you removed this function? It doesn't seem to be used anywhere:

``` bash
robin@rt480:~/Projects/anbox-cloud.io (fix-login) $ rg is_user_logged_in
webapp/app.py
62:    def is_user_logged_in(*args, **kwargs):
74:    return is_user_logged_in
```

Reply:
This is a function that is necessary to create the decorator: `@login_required` that we use for endpoints behind SSO

Reply:
Oh sorry, I see. My bad.

Reply:
No, I wrote a comment saying that when I remove it, it returns an error. The decorator must return a function